### PR TITLE
issue #116

### PR DESCRIPTION
**Pull request by Harold2017**
sorry, i raise a wrong pull request to master branch last time...

replace func.strftime with func.date
solve distinct query problem with MySQL

### CODE REVIEW

File: flask_monitoringdashboard/database/versions.py [hunk 1]
```diff
@@ -21,7 +21,8 @@ def get_versions(db_session, end=None):
     :param end: the versions that are used in a specific endpoint
     :return: a list with the versions (as a string)
     """
-    result = db_session.query(distinct(FunctionCall.version)). \
+    result = db_session.query(FunctionCall). \
```
Comment (bogdanp05) [inline flask_monitoringdashboard/database/versions.py]: This method now fails a test

**Comment by bogdanp05** [comment]
Hi @Harold2017 ! The first change you implemented looks fine, but the second one causes a failed test. You can do another commit to just keep the first change and then I can approve the pull request.

File: flask_monitoringdashboard/database/versions.py (hunk 1 above)
Comment (Harold2017) [reply flask_monitoringdashboard/database/versions.py]: Hi, i think it should be OK to pass the unittest now XD

**Comment by bogdanp05** [comment]
When trying to use MySql, `get_version` method fails with:
```
sqlalchemy.exc.InternalError: (pymysql.err.InternalError) (1055, "Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'flask_test.functionCalls.id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by") [SQL: 'SELECT DISTINCT `functionCalls`.id AS `functionCalls_id`, `functionCalls`.endpoint AS `functionCalls_endpoint`, `functionCalls`.execution_time AS `functionCalls_execution_time`, `functionCalls`.time AS `functionCalls_time`, `functionCalls`.version AS `functionCalls_version`, `functionCalls`.group_by AS `functionCalls_group_by`, `functionCalls`.ip AS `functionCalls_ip` \nFROM `functionCalls` \nWHERE true = 1 GROUP BY `functionCalls`.version ORDER BY `functionCalls`.time ASC']
```

File: flask_monitoringdashboard/test/constants.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Contains a number of constants that are used throughout the project
```
Comment (bogdanp05) [inline flask_monitoringdashboard/test/constants.py]: This file should not exist.

**Comment by bogdanp05** [comment]
Looks fine now!

**Pull request merged**